lang/en.json

```
@@ -863,6 +863,8 @@
 "DND5E.ItemActivationType": "Activation Type",
 "DND5E.ItemAmmoProperties": "Ammunition Properties",
 "DND5E.ItemAttackBonus": "Attack Roll Bonus",
+"DND5E.ItemAttackFlat": "Flat Bonus",
+"DND5E.ItemAttackFlatHint": "If checked, all bonuses except the value input here will be ignored.",
```

Probably need to make this a bit more explicit as excluding proficiency and ability modifiers:

```suggestion
"DND5E.ItemAttackFlatHint": "If checked, don't add proficiency, ability modifier, or any other bonuses to attack roll except what is included here.",
```